tests/unittest/_torch/auto_deploy/unit/utils/test_e8m0.py

```
@@ -0,0 +1,155 @@
+        actual,
+        torch.tensor([0.25, 1.5, 8.0], dtype=torch.float32, device="cpu"),
+        rtol=0,
+        atol=0,
+    )
+
+
+@_requires_e8m0
+def test_e8m0_helpers_reject_non_e8m0_inputs() -> None:
+    scale = torch.ones(2, dtype=torch.float32, device="cpu")
+
+    with pytest.raises(TypeError, match="expected an torch.float8_e8m0fnu tensor"):
```

Ruff (0.15.11)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)